pandas/tests/reshape/test_concat.py

```diff
@@ -2350,6 +2350,21 @@ def test_concat_datetime_timezone(self):
 
         tm.assert_frame_equal(result, expected)
 
+    def test_concat_series_name_npscalar_tuple(self):
```

Comment:
this doesn't need to be in the class, it can just be a module level test (e.g. de-dent)